x-pack/plugins/spaces/server/lib/toggle_ui_capabilities.ts

```diff
@@ -60,6 +60,21 @@ function toggleDisabledFeatures(
       });
     });
 
+    // TODO: Revisit if/when savedObjectsManagement UI Capabilities are refactored
```

Comment:
Note: I didn't add specific unit tests to [`toggle_ui_capabilities.test.ts`](https://github.com/legrego/kibana/blob/80aae6bba81f29a1a2533087ecca2003666fc6d0/x-pack/plugins/spaces/server/lib/toggle_ui_capabilities.test.ts) because this is covered by the [`ui_capabilities/spaces_only/saved_objects_management.ts`](https://github.com/legrego/kibana/blob/0a85af08bf477ac017b50f97bee61a3a24993608/x-pack/test/ui_capabilities/spaces_only/tests/saved_objects_management.ts) test suite, and that seemed sufficient given the hopefully temporary nature of this change